donate/templates/pages/core/contributor_support_page.html

```diff
@@ -0,0 +1,84 @@
+{% extends "pages/base_page.html" %}
+{% load form_tags static wagtailcore_tags wagtailimages_tags i18n %}
+
+{% block content %}
+
+<header class="page-header">
+    <div class="page-header__container">
+        <h1 class="page-header__heading">{{ page.title }}</h1>
+    </div>
+</header>
+
+<div class="layout">
+    <div class="layout__container">
+        <div class="layout__full-col">
+            <div class="page-body">
+                <p>{% blocktrans %}
+                    Here is some intro guide text to explain what this form is for and who should be filling it out.
+                {% endblocktrans %}</p>
+
+                <form class="support-form action="https://mozillasalescloud--mkl.cs4.my.salesforce.com/servlet/servlet.WebToCase?encoding=UTF-8" method="POST">
+
+                    <input type=hidden name="orgid" value="{{ orgid }}">
+
+                    {# TODO: should the retURL be a static "thank you for your question" page rather than "the current page"? #}
+                    <input type=hidden name="retURL" value="{{ page.full_url }}">
+
+                    <fieldset class="form-item">
+                        <label for="name">{% trans "Contact Name" %}</label>
+                        <input id="name" maxlength="80" name="name" size="20" type="text">
+                    </fieldset>
+
+                    <fieldset class="form-item">
+                        <label for="email">{% trans "Email" %}</label>
+                        <input id="email" maxlength="80" name="email" size="20" type="text">
+                    </fieldset>
+
+                    <fieldset class="form-item">
+                        <label for="00N0B000006X3fw">{% trans "Language" %}</label>
+                        <select id="00N0B000006X3fw" name="00N0B000006X3fw" title="Language">
```

Comment:
@alanmoo or @stephaniemcv, is this value related to anything we can set? Or is this just the id that salesforce will always use for a language picker?